### DISCUSSION TITLE
whats --add-host=host.docker.internal:host-gateway for?

### DISCUSSION THREAD

**Discussion by Jannik44** [nextcloud/all-in-one#3963] [Questions] [answered]
Hello, i read the documentation regarding proxy setup and came across the following, my question is now, why do we need that in proxy configuration? i mean instead of using the host network couldnt we just use another docker network? why is this step important or is it as important?

`⚠️ Important: In order to make this work on Docker for Linux, you need to add --add-host=host.docker.internal:host-gateway to the docker run command of your reverse proxy container or extra_hosts: ["host.docker.internal:host-gateway"] in docker compose (it works on Docker Desktop by default).

Another option and actually the recommended way in this case is to use --network host option (or network_mode: host for docker-compose) as setting for the reverse proxy container to connect it to the host network. If you are using a firewall on the server, you need to open ports 80 and 443 for the reverse proxy manually. By doing so, the default sample configurations that point at localhost:$APACHE_PORT should work without having to modify them.`

**Comment by szaimen** [comment, marked as answer]
Hi, I found this: https://www.google.com/search?q=%22--add-host%3Dhost.docker.internal%3Ahost-gateway%22
